src/pages/Layout/Layout.tsx

```diff
@@ -1,9 +1,31 @@
+import { HiOutlineHome } from 'react-icons/hi2';
+import { HiOutlineChatBubbleBottomCenterText } from 'react-icons/hi2';
+import { HiOutlineUser } from 'react-icons/hi2';
+import { HiOutlineUserGroup } from 'react-icons/hi2';
+import { HiOutlineUserPlus } from 'react-icons/hi2';
+import { HiOutlineCog6Tooth } from 'react-icons/hi2';
 import { Outlet } from 'react-router-dom';
+import Navigation from 'src/components/Navigation/Navigation';
+import Sidebar from 'src/components/Sidebar/Sidebar';
 import Container from 'src/components/core/Container/Container';
 
+const navItems = [
+  { name: 'Feed', path: '/home', icon: HiOutlineHome },
+  { name: 'Messages', path: '/messages', icon: HiOutlineChatBubbleBottomCenterText },
+  { name: 'Profile', path: '/profile', icon: HiOutlineUser },
+  { name: 'Friends', path: '/friends', icon: HiOutlineUserGroup },
+  { name: 'Requests', path: '/requests', icon: HiOutlineUserPlus },
+  { name: 'Settings', path: '/settings', icon: HiOutlineCog6Tooth }
+];
```

Comment:
I would put this in separate file in utils folder it is not really part of Layout